tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -507,7 +507,8 @@ lldbg_create_subprogram_mdnode(
           ll_get_md_node(db->module, LL_PlainMDNode, &lv_list_mdnode, 1));
     else
       llmd_add_md(mdb, lv_list_mdnode);
```

Comment:
Apparently, this was already adding `lv_list_mdnode` for LLVM versions between 3.4 and 7.0, but in the corresponding `DISubprogram` templates (in `ll_write.cpp`), the name of the field was `variables` instead of `retainedNodes`.

Reply:
Agree. Note that what was in the past stays in the past, I don't want to complicate the logic of this code by touching paths that deal with the backward compatibility.

Reply:
No objection.